libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -334,7 +334,7 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
 
     float current, consumed_mah, consumed_wh;
     const int8_t percentage = battery_remaining_pct(instance);
-    
+
```

Comment:
Please remove the whitespace changes.

Fix the whitespace problems as you change the code.

Reply:
Done.